### PR TITLE
Pk/upstream changes to push

### PR DESCRIPTION
**Pull request by PradeepKadubandi**
@affandar /  @simonporter ,

What do you think about this change? I made this in my fork for service fabric custom provider so that I could reuse the test orchestration code, I would like to push these changes to main repository so that there will be less conflicts or changes in future for pull from remote. I think the change is good and no risk.

Let me know if you have any comments and I can fix it.

### CODE REVIEW

File: Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj [hunk 1]
```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (simonporter) [inline Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj]: Add this import <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory),DurableTask.sln))\tools\DurableTask.props" />

File: Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{2BC2A5AD-4200-41B9-A034-332BB1D7187B}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>DurableTask.Test.Orchestrations</RootNamespace>
+    <AssemblyName>DurableTask.Test.Orchestrations</AssemblyName>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```
Comment (simonporter) [inline Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj]: Once you have the above import you can remove this then all projects will be aligned on version

**Comment by simonporter** [comment]
Added two minor comments if you can update I'll merge them in.

File: Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj (hunk 1 above)
Comment (PradeepKadubandi) [inline Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj]: Done.

File: Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj (hunk 2 above)
Comment (PradeepKadubandi) [inline Test/DurableTask.Test.Orchestrations/DurableTask.Test.Orchestrations.csproj]: Done.

**Comment by PradeepKadubandi** [comment]
@simonporter , Took care of the comments - can you merge the changes? Thanks!

**Pull request merged**